plugins/agent_types/AcquiaLiftAgent.inc

```diff
@@ -532,20 +477,6 @@ class AcquiaLiftAgent extends PersonalizeAgentBase implements PersonalizeAgentGo
   }
 
   /**
-   * Implements PersonalizeExplicitTargetingInterface::explicitTargetingSupportMultiple().
-   */
-  public static function explicitTargetingSupportMultiple() {
-    return PersonalizeExplicitTargetingInterface::EXPLICIT_TARGETING_MULTIPLE_BOTH;
-  }
-
-  /**
```

Comment:
Heh, well there kind of is still an open question as to whether we should just rip out support for auto-targeting now, but I didn't mean to go ahead and do that just yet so this is an oversight. Thanks for catching! :)